tools/docker-compose-base.yml

```diff
@@ -47,7 +47,7 @@ services:
       CACHED_ASSETS_S3_ACCESS_KEY_ID: ${CACHED_ASSETS_S3_ACCESS_KEY_ID-}
       CACHED_ASSETS_S3_SECRET_ACCESS_KEY: ${CACHED_ASSETS_S3_SECRET_ACCESS_KEY-}
       CACHED_ASSETS_S3_REGION: ${CACHED_ASSETS_S3_REGION-us-east-1}
```

Comment:
the above variables don't exist anymore, if I understand correctly.

Reply:
Thanks! I removed them.